src/probeinterface/wiring.py

```diff
@@ -41,7 +41,16 @@
         11, 8, 9, 10, 12, 7, 6, 5, 4, 3, 13, 2, 1, 14, 0, 15,
         55, 56, 54, 57, 53, 58, 52, 59, 51, 60, 50, 61, 62, 49, 63, 48],
 
-    # Done by Zach McKenzie & Chris Halcrow
+    # done by Bao Le, verified by Chris Halcrow
```

Comment:
```suggestion
    # done by Bao Le, verified by Alessio Buccino
```
I had some time and checked this. Looks correct :)